src/run.js

```diff
@@ -3,6 +3,7 @@ var grammar = require('./grammar');
 var interp = require('./interp');
 
 export function run(code, options) {
+  code = code += '\n'
```

Comment:
Uh.. Shouldn't this be

```
code += '\n'
```

or

```
code = code + '\n'
```

?

Also you're missing a semicolon there (oh the irony)